jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java

```diff
@@ -191,6 +192,18 @@ public JavaDockerContextGenerator setEntrypoint(List<String> entrypoint) {
     return this;
   }
 
+  /**
+   * Sets the user name (or UID) and optionally the user group (or GID) which the process in the
```

Comment:
Wording suggestion: `Sets the user for the {@code USER} directive.`

Reply:
done